src/services/EdFi.DataManagementService.Api.Tests.E2E/Management/ContainerSetup.cs

```diff
@@ -28,4 +28,32 @@ public async Task<string> SetupDataManagement()
             dockerImage.GetMappedPublicPort(8080)
         ).ToString();
     }
+
+    public async Task<string> SetupDataBase(string engine)
+    {
+        string imageName = string.Empty;
+        ushort port = 0;
+
+        if (engine == "postgresql")
+        {
+            imageName = "local/edfi-data-management-postgresql";
+            port = 5432;
+        }
+
+        var dockerImage = new ContainerBuilder()
+            .WithImage(imageName)
+            .WithPortBinding(port, port)
+            .WithWaitStrategy(Wait.ForUnixContainer().UntilPortIsAvailable(port))
```

Comment:
When I run locally I have to remove this line, otherwise the tests never start. Maybe I have something misconfigured?